docs/DifferentiableProgramming.md

```diff
@@ -174,9 +174,9 @@ third-party libraries like [TensorFlow](https://github.com/tensorflow/swift).
 import TensorFlow
 
 let model = Sequential {
-    var layer1 = Dense<Float>(inputSize: 784, outputSize: 100, activation: relu)
-    var layer2 = Dense<Float>(inputSize: 100, outputSize: 30, activation: relu)
-    var layer3 = Dense<Float>(inputSize: 30, outputSize: 3, activation: identity)
+    Dense<Float>(inputSize: 784, outputSize: 100, activation: relu)
```

Comment:
The old `Sequential` function builder code didn't compile:
```console
$ cat sequential.swift
import TensorFlow

let model = Sequential {
    var dense1 = Dense<Float>(inputSize: 784, outputSize: 100, activation: relu)
    var dense2 = Dense<Float>(inputSize: 100, outputSize: 30, activation: relu)
    var dense3 = Dense<Float>(inputSize: 30, outputSize: 3, activation: identity)
}

$ swift sequential.swift
sequential.swift:3:24: error: cannot invoke 'buildBlock' with no arguments
let model = Sequential {
                       ^
sequential.swift:3:24: note: overloads for 'buildBlock' exist with these partially matching parameter lists: (L1, L2), (L1, L2, L3), (L1, L2, L3, L4), (L1, L2, L3, L4, L5), (L1, L2, L3, L4, L5, L6), (L1, L2, L3, L4, L5, L6, L7), (L1, L2, L3, L4, L5, L6, L7, L8), (L1, L2, L3, L4, L5, L6, L7, L8, L9), (L1, L2, L3, L4, L5, L6, L7, L8, L9, L10)
let model = Sequential {
                       ^
```

I'm pretty sure it was a copy-paste error from other code.